larray/core.py

```diff
@@ -8761,6 +8774,10 @@ def split_axis(self, axis, sep='_', names=None):
         names : list of names, optional
             names of resulting axes. Defaults to split the combined axis name
             using the given delimiter string.
+        regex : str, optional
+            use regex instead of simple delimiter to split the axis.
+            Names of new axes must be provided.
```

Comment:
We could possibly lift the names must be provided restriction, as there could be case were the normal sep is valid for the axis name but not for the axis labels (such as in the example).